### PR TITLE
fix styling

### PR DESCRIPTION
**Pull request by d3ns0n**
I fixed some styling issues like missing semicolons, double quotes and missing empty lines at EOF

### CODE REVIEW

File: src/app/menu/menu.component.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import { Component } from '@angular/core';
 
 @Component({
-  selector: 'menu',
+  selector: 'app-menu',
```
Comment (wassertim) [inline src/app/menu/menu.component.ts]: nice change :)

File: src/app/todo/services/task-result.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import { Task } from "./task";
+import { Task } from './task';
 
 export interface TaskResult {
-    content: Array<Task>
-}
\ No newline at end of file
+  content: Array<Task>;
```
Comment (wassertim) [inline src/app/todo/services/task-result.ts]: 👍

File: src/app/todo/services/todo.service.ts [hunk 3]
```diff
@@ -2,18 +2,18 @@ import { Injectable } from '@angular/core';
...
-import 'rxjs/add/operator/map'
+import 'rxjs/add/operator/map';
 
 @Injectable()
 export class TodoService {
 
-    constructor(private http: HttpClient) {}
+  constructor(private http: HttpClient) { }
 
     list(): Observable<Array<Task>> {
```
Comment (wassertim) [inline src/app/todo/services/todo.service.ts]: why did not you indent this?

File: src/app/todo/todo-list.component.ts [hunk 4]
```diff
@@ -11,9 +11,9 @@ export class TodoListComponent {
 
   constructor(private todoService: TodoService) {
     console.log('call the service');
```
Comment (wassertim) [inline src/app/todo/todo-list.component.ts]: we can remove this

File: src/app/menu/menu.component.ts (hunk 1 above)
Comment (d3ns0n) [reply src/app/menu/menu.component.ts]: yeah TSLint is quite a cool Plugin for VSC

File: src/app/todo/services/todo.service.ts (hunk 3 above)
Comment (d3ns0n) [reply src/app/todo/services/todo.service.ts]: no idea what happened. will fix it.

**Pull request merged**